src/main/java/jwbot/command/AddEventCommand.java

```diff
@@ -0,0 +1,45 @@
+package jwbot.command;
+
+import jwbot.data.TaskList;
+import jwbot.data.exception.JwBotException;
+import jwbot.data.task.Event;
+import jwbot.storage.Storage;
+import jwbot.ui.Ui;
+
+public class AddEventCommand extends Command {
+
+    public AddEventCommand(String input) {
+        super(input);
+    }
+
+    /**
+     * The method that executes the command
+     *
+     * @param tasks list of the tasks
+     * @param ui the ui object responsible for user interaction
+     * @param storage the storage object responsible for writing and reading txt file
+     */
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) throws JwBotException {
+        try {
+            String content = input.split(" ", 2)[1];
+            String[] separated = content.split(" /at ");
```

Comment:
Nice that you followed the coding standard and used String[] separated instead of String separated[] ! 👍